versionedRouter.js

```diff
@@ -94,14 +103,47 @@ async function handleDest(ctx, version, destination) {
     ...metaTags
   });
   const respList = [];
+  const executeStartTime = new Date();
   await Promise.all(
     events.map(async event => {
       try {
         let parsedEvent = event;
         parsedEvent.request = { query: reqParams };
         parsedEvent = processDynamicConfig(parsedEvent);
-        let respEvents = await destHandler.process(parsedEvent);
-
+        let respEvents;
+        if (isCdkDestination(destination)) {
+          const cdkResponse = await Executor.execute(
+            event,
+            ConfigFactory.getConfig(destination)
+          );
+          respEvents = await destHandler.process(parsedEvent);
```

Comment:
You can put this below Line 113 & delete the else block

Reply:
done